ert_shared/plugins/hook_specifications/jobs.py

```diff
@@ -33,3 +33,11 @@ def installable_workflow_jobs():
     :return: dict with workflow job names as keys and path to config as value
     :rtype: PluginResponse with data as dict[str,str]
     """
+
+
+@hook_specification
+def legacy_ertscript_workflow(config):
+    """
+    :param config:
```

Comment:
Would be nice with some documentation here (at least the type of the config). Would also be nice to add a section about the new hook in the rst documentation!

Reply:
Will do, do you agree with the overall implementation though? DIdnt want to spend to much time writing documentation if you fundamentally disagreed.

Reply:
Yes I do! :+1: